config/locales/fr.yml

```
@@ -6,6 +6,7 @@ fr:
   api_entreprise:
     default_error_message:
       etablissement: Erreur lors de la récupération des détails de l’établissement.
+    generic_error: Nous n'avons pu récupérer les détails de l'entreprise auprès de nos partenaires, veuillez réessayer ultérieurement
```

```suggestion
    generic_error: Nous n’avons pu récupérer les détails de l’entreprise auprès de nos partenaires, veuillez réessayer ultérieurement
```
Je sais pas si c'est très utile